src/components/TodoFilters.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { FILTER_TYPES } from '../App';
+
+const TodoFilters = ({ selectedFilterItem, setItemFilter }) => (
+  <ul className="filters">
+    <li>
+      <a
+        href="#/"
+        // работа с classnames через cn
+        className={cn({ selected: selectedFilterItem === 'all' })}
+        onClick={() => setItemFilter(FILTER_TYPES.all)}
+      >
+              All
+      </a>
+    </li>
+
+    <li>
+      <a
+        href="#/active"
+        className={selectedFilterItem === 'active' ? 'selected' : ''}
```

Comment:
Используй везде cn, раз уж использовала в одном месте
```suggestion
        className={cn({ selected: selectedFilterItem === 'FILTER_TYPES.active' })}
```